src/lightkurve/io/eleanorlite.py

```diff
@@ -0,0 +1,112 @@
+"""Reader for GSFC-ELEANOR-LITE light curve files. 
+Details can be found at https://archive.stsci.edu/hlsp/eleanor and https://archive.stsci.edu/hlsp/gsfc-eleanor-lite
+"""
+from ..lightcurve import TessLightCurve
+from ..utils import TessQualityFlags
+from astropy import units as u
+
+from .generic import read_generic_lightcurve
+
+import numpy as np
+
+def read_eleanorlite_lightcurve(filename,
+    flux_column="CORR_FLUX",
+    quality_bitmask="default", 
+    **kwargs):
+    """Returns a `TessLightCurve` object given a light curve file from the GSFC Eleanor-lite Pipeline (https://archive.stsci.edu/hlsp/gsfc-eleanor-lite).
+
+    By default, eleanor's `CORR_FLUX` column is used to populate the `flux` values. Note that the "FLUX_ERR" column in the 
+    Eleanor FITS file is referred to the uncertainty of "RAW_FLUX", not "CORR_FLUX". 
+    Thus the uncertainty reported in the 'flux_err' column here is calculated as follows: corr_flux_err = corr_flux*raw_flux_err/raw_flux. 
+    For completeness, the original raw_flux's error is added as a "raw_flux_err" column.
+    In terms of quality flags, eleanor uses the TESS SPOC quality flags by identifying short-cadence targets that fall on each camera-CCD pairing for a given sector. However, eleanor, also adds two new quality flags -- bit 17 (decimal value 131072)) and bit 18 (decimal value 262144). 
+
+    Parameters
+    ----------
+    filename : str
+        Local path or remote url of a GSFC-ELEANOR-LITE light curve FITS file.
+    flux_column : 'RAW_FLUX', 'CORR_FLUX', 'PCA_FLUX', or 'FLUX_BKG'
+        Which column in the FITS file contains the preferred flux data?
+        By default the "Corrected Flux" flux (CORR_FLUX) is used.
+    flux_err_column: 'FLUX_ERR'
```

Comment:
`flux_err_column` is no longer a parameter. So it should be removed from Parameters section of the docstring.